

*
* Visual Studio Code Server
*
* Reminder: You may only use this software with Visual Studio family products,
* as described in the license https://aka.ms/vscode-remote/license
*

Extension host agent listening on 44507

[11:57:44] Extension host agent started.
[11:57:45] [127.0.0.1][4379045a][ManagementConnection] New connection established.
[11:57:45] [127.0.0.1][565ca51b][ExtensionHostConnection] New connection established.
[11:57:45] [127.0.0.1][565ca51b][ExtensionHostConnection] <1377> Launched Extension Host Process.
[11:57:45] Deleted from disk ms-toolsai.jupyter /home/webalive/.vscode-server/extensions/ms-toolsai.jupyter-2021.8.2041215044
[11:57:46] Deleted from disk ms-python.python /home/webalive/.vscode-server/extensions/ms-python.python-2021.10.1317843341
[11:57:49] [File Watcher (chokidar)] Inotify limit reached (ENOSPC)
[11:57:53] Installing extension: ms-python.vscode-pylance
[11:57:53] Installing extension: ms-toolsai.jupyter
[11:57:57] Downloaded extension: ms-toolsai.jupyter /home/webalive/.vscode-server/data/CachedExtensionVSIXs/ms-toolsai.jupyter-2021.9.1101343141
[11:57:57] Downloaded extension: ms-python.vscode-pylance /home/webalive/.vscode-server/data/CachedExtensionVSIXs/ms-python.vscode-pylance-2021.10.1
[11:57:58] Extracted extension to /home/webalive/.vscode-server/extensions/.a68856d8-77f1-466c-beb3-81d51927a19c: ms-toolsai.jupyter
[11:57:58] Renamed to /home/webalive/.vscode-server/extensions/ms-toolsai.jupyter-2021.9.1101343141
[11:57:58] Extracting completed. ms-toolsai.jupyter
[11:57:58] Extension installed successfully: ms-toolsai.jupyter
[11:58:04] Extracted extension to /home/webalive/.vscode-server/extensions/.8f51d85a-bc3b-4bcf-b2c3-1716596b5389: ms-python.vscode-pylance
[11:58:04] Renamed to /home/webalive/.vscode-server/extensions/ms-python.vscode-pylance-2021.10.1
[11:58:04] Extracting completed. ms-python.vscode-pylance
[11:58:04] Extension installed successfully: ms-python.vscode-pylance
[12:02:32] [127.0.0.1][4379045a][ManagementConnection] The client has reconnected.
[12:02:32] [127.0.0.1][565ca51b][ExtensionHostConnection] The client has reconnected.
rejected promise not handled within 1 second: Error: Could not find pty with id "3"
stack trace: Error: Could not find pty with id "3",    at M._throwIfNoPty (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:17:5468),    at M.updateTitle (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:17:2172),    at Object.call (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:13:8402),    at f.onPromise (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:12:4940),    at f.onRawMessage (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:12:4299),    at /home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:12:3552,    at D.fire (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:8:1712),    at process.S (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:6:32863),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)
[Error: Could not find pty with id "3",    at M._throwIfNoPty (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:17:5468),    at M.updateTitle (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:17:2172),    at Object.call (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:13:8402),    at f.onPromise (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:12:4940),    at f.onRawMessage (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:12:4299),    at /home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:12:3552,    at D.fire (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:8:1712),    at process.S (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:6:32863),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)]
rejected promise not handled within 1 second: Error: Could not find pty with id "3"
stack trace: Error: Could not find pty with id "3",    at M._throwIfNoPty (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:17:5468),    at M.updateIcon (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:17:2232),    at Object.call (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:13:8402),    at f.onPromise (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:12:4940),    at f.onRawMessage (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:12:4299),    at /home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:12:3552,    at D.fire (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:8:1712),    at process.S (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:6:32863),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)
[Error: Could not find pty with id "3",    at M._throwIfNoPty (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:17:5468),    at M.updateIcon (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:17:2232),    at Object.call (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:13:8402),    at f.onPromise (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:12:4940),    at f.onRawMessage (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:12:4299),    at /home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:12:3552,    at D.fire (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:8:1712),    at process.S (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:6:32863),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)]
[12:05:15] [127.0.0.1][daca6404][ManagementConnection] New connection established.
[12:05:15] [127.0.0.1][1e27dfb5][ExtensionHostConnection] New connection established.
[12:05:15] [127.0.0.1][1e27dfb5][ExtensionHostConnection] <2057> Launched Extension Host Process.
[12:05:18] [File Watcher (chokidar)] Inotify limit reached (ENOSPC)
[IPC Library: Pty Host]  INFO Persistent process "5": Replaying 45 chars and 1 size events
[12:06:31] [127.0.0.1][daca6404][ManagementConnection] The client has reconnected.
rejected promise not handled within 1 second: Error: Could not find pty with id "6"
stack trace: Error: Could not find pty with id "6",    at M._throwIfNoPty (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:17:5468),    at M.updateTitle (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:17:2172),    at Object.call (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:13:8402),    at f.onPromise (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:12:4940),    at f.onRawMessage (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:12:4299),    at /home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:12:3552,    at D.fire (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:8:1712),    at process.S (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:6:32863),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)
[Error: Could not find pty with id "6",    at M._throwIfNoPty (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:17:5468),    at M.updateTitle (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:17:2172),    at Object.call (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:13:8402),    at f.onPromise (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:12:4940),    at f.onRawMessage (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:12:4299),    at /home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:12:3552,    at D.fire (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:8:1712),    at process.S (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:6:32863),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)]
rejected promise not handled within 1 second: Error: Could not find pty with id "6"
stack trace: Error: Could not find pty with id "6",    at M._throwIfNoPty (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:17:5468),    at M.updateIcon (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:17:2232),    at Object.call (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:13:8402),    at f.onPromise (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:12:4940),    at f.onRawMessage (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:12:4299),    at /home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:12:3552,    at D.fire (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:8:1712),    at process.S (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:6:32863),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)
[Error: Could not find pty with id "6",    at M._throwIfNoPty (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:17:5468),    at M.updateIcon (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:17:2232),    at Object.call (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:13:8402),    at f.onPromise (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:12:4940),    at f.onRawMessage (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:12:4299),    at /home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:12:3552,    at D.fire (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:8:1712),    at process.S (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:6:32863),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)]
[12:06:58] [127.0.0.1][daca6404][ManagementConnection] The client has reconnected.
[12:06:59] [127.0.0.1][1e27dfb5][ExtensionHostConnection] The client has reconnected.
rejected promise not handled within 1 second: Error: Could not find pty with id "7"
stack trace: Error: Could not find pty with id "7",    at M._throwIfNoPty (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:17:5468),    at M.updateTitle (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:17:2172),    at Object.call (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:13:8402),    at f.onPromise (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:12:4940),    at f.onRawMessage (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:12:4299),    at /home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:12:3552,    at D.fire (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:8:1712),    at process.S (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:6:32863),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)
[Error: Could not find pty with id "7",    at M._throwIfNoPty (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:17:5468),    at M.updateTitle (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:17:2172),    at Object.call (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:13:8402),    at f.onPromise (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:12:4940),    at f.onRawMessage (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:12:4299),    at /home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:12:3552,    at D.fire (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:8:1712),    at process.S (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:6:32863),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)]
rejected promise not handled within 1 second: Error: Could not find pty with id "7"
stack trace: Error: Could not find pty with id "7",    at M._throwIfNoPty (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:17:5468),    at M.updateIcon (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:17:2232),    at Object.call (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:13:8402),    at f.onPromise (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:12:4940),    at f.onRawMessage (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:12:4299),    at /home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:12:3552,    at D.fire (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:8:1712),    at process.S (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:6:32863),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)
[Error: Could not find pty with id "7",    at M._throwIfNoPty (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:17:5468),    at M.updateIcon (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:17:2232),    at Object.call (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:13:8402),    at f.onPromise (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:12:4940),    at f.onRawMessage (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:12:4299),    at /home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:12:3552,    at D.fire (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:8:1712),    at process.S (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:6:32863),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)]
[12:20:15] [127.0.0.1][4379045a][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/bootstrap.js:1:370)
    at process.emit (events.js:315:20)
[14:40:35] [127.0.0.1][daca6404][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/bootstrap.js:1:370)
    at process.emit (events.js:315:20)
[15:20:24] [127.0.0.1][4379045a][ManagementConnection] The reconnection grace time of 3h has expired, so the connection will be disposed.
[15:24:53] [127.0.0.1][565ca51b][ExtensionHostConnection] <1377> Extension Host Process exited with code: 0, signal: null.
[17:09:04] [127.0.0.1][daca6404][ManagementConnection] Another client has connected, will shorten the wait for reconnection 5m before disposing...
[17:09:04] [127.0.0.1][466ffc4d][ManagementConnection] New connection established.
[17:09:04] [127.0.0.1][d96257be][ExtensionHostConnection] New connection established.
[17:09:04] [127.0.0.1][d96257be][ExtensionHostConnection] <7792> Launched Extension Host Process.
[17:10:00] [127.0.0.1][466ffc4d][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
[17:10:02] [127.0.0.1][d96257be][ExtensionHostConnection] <7792> Extension Host Process exited with code: 0, signal: null.
[17:10:08] [127.0.0.1][f7dbab15][ManagementConnection] New connection established.
[17:10:08] [127.0.0.1][b5dbf714][ExtensionHostConnection] New connection established.
[17:10:08] [127.0.0.1][b5dbf714][ExtensionHostConnection] <7987> Launched Extension Host Process.
[IPC Library: Pty Host]  INFO Persistent process "5": Replaying 155 chars and 1 size events
[17:10:11] [File Watcher (chokidar)] Inotify limit reached (ENOSPC)
[17:14:04] [127.0.0.1][daca6404][ManagementConnection] The reconnection short grace time of 5m has expired, so the connection will be disposed.
[17:14:06] [127.0.0.1][1e27dfb5][ExtensionHostConnection] <2057> Extension Host Process exited with code: 0, signal: null.
[17:55:47] [127.0.0.1][b5dbf714][ExtensionHostConnection] The client has reconnected.
rejected promise not handled within 1 second: Error: Could not find pty with id "3"
stack trace: Error: Could not find pty with id "3",    at M._throwIfNoPty (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:17:5468),    at M.updateTitle (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:17:2172),    at Object.call (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:13:8402),    at f.onPromise (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:12:4940),    at f.onRawMessage (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:12:4299),    at /home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:12:3552,    at D.fire (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:8:1712),    at process.S (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:6:32863),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)
[Error: Could not find pty with id "3",    at M._throwIfNoPty (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:17:5468),    at M.updateTitle (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:17:2172),    at Object.call (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:13:8402),    at f.onPromise (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:12:4940),    at f.onRawMessage (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:12:4299),    at /home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:12:3552,    at D.fire (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:8:1712),    at process.S (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:6:32863),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)]
rejected promise not handled within 1 second: Error: Could not find pty with id "3"
stack trace: Error: Could not find pty with id "3",    at M._throwIfNoPty (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:17:5468),    at M.updateIcon (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:17:2232),    at Object.call (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:13:8402),    at f.onPromise (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:12:4940),    at f.onRawMessage (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:12:4299),    at /home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:12:3552,    at D.fire (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:8:1712),    at process.S (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:6:32863),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)
[Error: Could not find pty with id "3",    at M._throwIfNoPty (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:17:5468),    at M.updateIcon (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:17:2232),    at Object.call (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:13:8402),    at f.onPromise (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:12:4940),    at f.onRawMessage (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:12:4299),    at /home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:12:3552,    at D.fire (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:8:1712),    at process.S (/home/webalive/.vscode-server/bin/ee8c7def80afc00dd6e593ef12f37756d8f504ea/out/vs/platform/terminal/node/ptyHostMain.js:6:32863),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)]
[17:55:50] [127.0.0.1][f7dbab15][ManagementConnection] The client has reconnected.
